src/plugins/qml/MuseScore/Plugins/PluginsPage.qml

```diff
@@ -102,9 +102,7 @@ Item {
 
         anchors.top: parent.top
         anchors.left: parent.left
-        anchors.leftMargin: root.sideMargin
```

Comment:
It is better to return as it was. Like in scores and learn pages

Reply:
The change is necessary because otherwise the navigation/selection border would be cut off. We can't set `clip` to `false`, because that will result in the items being visible above and below the scrollable area.

(In the current master branch, the clipping problem does not occur, because the items have an inset of 4px. But that way, they are not aligned with the "Enabled"/"Disabled" title text, which looks a bit ugly. )